python/revng/internal/cli/support.py

```diff
@@ -88,7 +88,7 @@ def popen(command, options: Options, environment: OptionalEnv = None, **kwargs)
     return Popen(command, env=environment, **kwargs)
 
 
-def try_run(command, options: Options, environment: OptionalEnv = None) -> int:
+def try_run(command, options: Options, environment: OptionalEnv = None, **kwargs) -> int:
```

Comment:
>  What's the specific need for this?

Specifically I needed to pass `stdout=DEVNULL, stderr=DEVNULL` to an inkscape invocation

> We've been actively trying to keep the surface to spawn new processes small and uniform otherwise things becomes a mass real quick.

`revng.internal.cli.support.popen` has kwargs, I don't see a reason why its wrappers (run, try_run) should not.

Reply:
I was under the impression that all subprocess invocations had to go through `support.run` & friends. If that's not the case then I'm 100% ok with using `subprocess.run`

Reply:
Dropped use of `support.run` from `mass-testing`.